components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -243,20 +245,32 @@ const ContributorProfileHeader = ({
             <DialogTitle className="!text-3xl text-left">Collaborate with {username}!</DialogTitle>
           </DialogHeader>
 
-          <form onSubmit={handleCollaborationRequest} className="flex flex-col w-full gap-4 px-4 md:gap-8 md:px-14 ">
+          <form onSubmit={handleCollaborationRequest} className="flex flex-col w-full gap-2 px-4 md:gap-8 md:px-14 ">
             <div className="space-y-2">
               <label htmlFor="message">Send a message</label>
               <Textarea
                 defaultRow={1}
                 value={message}
-                className="w-full px-2 mb-2 transition text-light-slate-11 focus:outline-none"
+                className={clsx(
+                  "w-full px-2 mb-2 transition text-light-slate-11 focus:outline-none border",
+                  message.length > 0 && message.length < 20 && "border-red-500"
+                )}
                 name="message"
                 id="message"
                 onChangeText={handleTextAreaInputChange}
               />
+              <Text small className="select-none text-light-slate-11">
```

Comment:
nit: we can remove the `Text` component entirely if the character is greater than 20 here

Move the login outside the `Text` tag 👌

Reply:
Very well, I should! There was a certain behavior I was tryna avoid with that approach yea... but it didn't avoid it still. Lazy to have just left it that way :rofl: